This (temporary) patch fixes some current isues with the E library
renaming.  It is expected that either OE upstream will fix this,
or that something in E upstream will fix this, so as soon as that
event happens, this should be removed.

hacked-by: Mike Westerhof <mwester@dls.net>

diff --git a/recipes/efl1/ecore.inc b/recipes/efl1/ecore.inc
index 2609414..1680937 100644
--- a/recipes/efl1/ecore.inc
+++ b/recipes/efl1/ecore.inc
@@ -27,8 +27,8 @@ PACKAGES += "\
     ${PN}-x \
     ${PN}-input \
     "
-FILES_${PN} = "${libdir}/libecore-*.so.*"
-FILES_${PN}-con = "${libdir}/libecore_con-*.so*"
+FILES_${PN} = "${libdir}/libecore.so.*"
+FILES_${PN}-con = "${libdir}/libecore_con.so*"
 FILES_${PN}-config = "${libdir}/libecore_config*.so*"
 FILES_${PN}-desktop = "${libdir}/libecore_desktop*.so*"
 FILES_${PN}-directfb = "${libdir}/libecore_directfb*.so*"
@@ -36,7 +36,7 @@ FILES_${PN}-sdl = "${libdir}/libecore_sdl*.so*"
 FILES_${PN}-evas = "${libdir}/libecore_evas*.so*"
 FILES_${PN}-fb = "${libdir}/libecore_fb*.so*"
 FILES_${PN}-file = "${libdir}/libecore_file*.so*"
-FILES_${PN}-imf = "${libdir}/libecore_imf-*.so.*"
+FILES_${PN}-imf = "${libdir}/libecore_imf.so.*"
 FILES_${PN}-imf-evas = "${libdir}/libecore_imf_evas*.so*"
 FILES_${PN}-ipc = "${libdir}/libecore_ipc*.so*"
 FILES_${PN}-job = "${libdir}/libecore_job*.so*"
